contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,408 @@
+
+```node
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'alias_id',
+})
+```
+
+```javascript
+posthog.alias('alias_id', 'distinct_id');
+```
+
+```python
+posthog.alias('distinct_id', 'alias_id', );
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id", "alias_id");
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "distinct_id",
+        "alias": "alias_id"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "event": "$create_alias"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+There are two requirements when assigning an `alias_id`:
+
+1. It cannot be associated with more than one `distinct_id`.
+2. The `alias_id` **must not** be the `distinct_id` of any other user. For example:
+
+<MultiLanguage selector="tabs">
+
+```node
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```javascript
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```python
+# ❌ Not possible
+posthog.alias('distinct_id_of_another_user', 'distinct_id');
+```
+
+```php
+# ❌ Not possible
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'distinct_id_of_another_user'
+));
+```
+
+```ruby
+# ❌ Not possible
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "distinct_id_of_another_user",
+})
+```
+
+```go
+// ❌ Not possible
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "distinct_id_of_another_user",
+})
+```
+
+```java
+// ❌ Not possible
+posthog.alias("distinct_id", "distinct_id_of_another_user");
+```
+
+</MultiLanguage>
+
+If we encounter an `identify` or `alias` event with one of the above problems, the following will happen:
+
+1. We process the event normally (it will be ingested and show up in the UI)
+2. Merging users will be refused and an ingestion warning will be logged (see [ingestion warnings](/manual/data-management#ingestion-warnings) for more details).
+3. The event will be only be tied to the user associated with `distinct_id`
+
+## Troubleshooting and FAQs
+
+### How are properties managed when identifying anonymous users?
+
+When an anonymous user is identified as `User A`, all the properties of the anonymous user are added to `User A`. If there is a conflict, the properties of `User A` are prioritized over the anonymous user. For example:
+
+```js
+client.identify({
+    distinctId: 'user_A',
+    properties: {
+        name: 'User A',
+        location: 'London',
+        timezone: 'GMT',
+    },
+})
+
+client.reset()
```

Comment:
I'd use for simplicity, so they don't need to go back understanding identify and reset
```suggestion
/* User A have properties:
{ 
        name: 'User A',
        location: 'London',
        timezone: 'GMT',
}
*/
```